docs/guide/setup-examples/julia.md

```diff
@@ -0,0 +1,401 @@
+---
+title: How to use Sublime for Julia development
+---
+
+Adapted from https://github.com/PetrKryslUCSD/HowToUseJuliaWithSublimeText3
+This guide was written by a Windows user,
+that uses the Windows Subsystem for Linux to run Julia.
+
+When working with Julia,
+I like having my source code on the left
+and a terminal running Julia REPL on the right.
+This setup will allow you to have the terminal inside Sublime
+and to have commands that send code under your cursor to the REPL.
+
+## Installation
+
+### Install Julia syntax highlighting
+
+Install the [Julia][] package.
+Bring up the **Command Palette**,
+and type in a few letters of **Package Control: Install Package**.
+When it comes up, select it.
+Loading the registry will take a couple of seconds,
+and then we get window where we can type in `Julia`.
+A button named **Julia**
+(with the subtitle "Julia syntax highlighting for Sublime Text 2/3")
+will come up highlighted.
+Click on it, and the package will be installed.
+At this point one should be able to open a Julia source file
+and get it highlighted based upon the syntax of Julia.
```

Comment:
As I mentioned in the other comment, we don't want to copy this section into every page that describes a workflow and depends on third-party packages, but for now you can leave it in and we remove it once such a page exists.